src/components/Verify.tsx

```diff
@@ -1,87 +1,79 @@
 import { useState } from 'react';
+import { useFieldArray, useForm } from 'react-hook-form';
+import { GrFormClose } from 'react-icons/gr';
 import type { Chain, Hash } from 'viem';
+import { isAddress } from 'viem';
+import FormErrorMessage from '@/components/ui/FormErrorMessage';
 import { SelectChain } from '@/components/ui/SelectChain';
 import { SUPPORTED_CHAINS } from '@/lib/constants';
 import type { BuildConfig, BuildFramework, VerifyData } from '@/lib/cove-api';
 
+type TxFormValues = {
+  repoUrl: string;
+  repoCommit: string;
+  contractAddress: string;
+  framework: string;
+  buildHint: string;
+  creationTxHashes: {
+    chainId: number;
+    hash: string;
+  }[];
+};
```

Comment:
can we use the API types and viem types for stronger typing here? Also `creationTxHashes` needs the chain's short name (e.g. `mainnet` and not the chain ID. This is an artifact of the ethers-rs types though I agree chain ID would be better, I can look into changing it later but it's not a priority for now)
```suggestion
type TxFormValues = {
  repoUrl: string;
  repoCommit: string;
  contractAddress: Address;
  framework: BuildFramework;
  buildHint: string;
  creationTxHashes: {
    chainId: string;
    hash: Hash;
  }[];
};
```

Reply:
I have everything but chainId up. I will update that tomorrow morning.

Reply:
https://github.com/ScopeLift/cove-frontend/pull/9/commits/6e9acb989cb45a75d0e033ae98d325b539f843f1